saltedge-compliance-connector-java-v2/compliance-connector-sdk/src/main/java/com/saltedge/connector/sdk/provider/ConnectorSDKCallbackService.java

```diff
@@ -187,13 +188,24 @@ public String onPaymentInitiationAuthorizationSuccess(
     Map<String, String> paymentExtraMap = parseExtra(paymentExtra);
 
     String sessionSecret = paymentExtraMap.get(SDKConstants.KEY_SESSION_SECRET);
-    String status = getStatusOfPaymentProduct(paymentProduct);
+    String status = getFinalStatusOfPaymentProduct(paymentProduct);
     SessionSuccessCallbackRequest params = new SessionSuccessCallbackRequest(userId, status);
     if (!StringUtils.isEmpty(sessionSecret)) sessionsCallbackService.sendSuccessCallback(sessionSecret, params);
 
     return paymentExtraMap.getOrDefault(SDKConstants.KEY_RETURN_TO_URL, "");
   }
 
+  @Override
+  public boolean updatePaymentFundsInformation(Boolean fundsAvailable, String paymentExtra, String paymentProduct) {
+    Map<String, String> paymentExtraMap = parseExtra(paymentExtra);
+    String sessionSecret = paymentExtraMap.get(SDKConstants.KEY_SESSION_SECRET);
+    String status = getFinalStatusOfPaymentProduct(paymentProduct);
```

Comment:
Funds confirmation stage is not final. Let ASPSP to provide status or use previous one.